Source/Scene/Cesium3DTileBatchTableResources.js

```diff
@@ -398,10 +404,13 @@ define([
         }
 
         var that = this;
-        return function(source) {
+        // TODO: remove handleTranslucent? need vector render order
```

Comment:
We'll have a better answer when we connect declarative styling to this, but we may be able to remove it when we have opaque and transparent ground passes.